themes/socialbase/templates/layout/page.html.twig

```diff
@@ -39,16 +39,20 @@
 
 <main id="content" class="main-container" role="main">
 
-  {% if page.breadcrumb %}
-    {{ page.breadcrumb }}
+  {% if not style == 'sky' %}
+    {% if page.breadcrumb %}
+      {{ page.breadcrumb }}
+    {% endif %}
```

Comment:
Socialbase shouldn't be aware of the `sky` flavour. Perhaps we can remove the breadcrumbs through a preprocess hook in the `socialblue` theme. 

@xinyuma should also probably think about how we want to use breadcrumbs throughout all flavours, I think this is still something we're inconsistent with even though they can be useful.